src/states/sidemenu/controller.js

```diff
@@ -4,43 +4,40 @@
 
     .controller('SidemenuController', SidemenuController);
 
-  function SidemenuController($scope, $ionicModal, $timeout) {
+  SidemenuController.$inject = ['$scope', '$ionicModal', '$timeout'];
 
-    // With the new view caching in Ionic, Controllers are only called
-    // when they are recreated or on app start, instead of every page change.
-    // To listen for when this page is active (for example, to refresh data),
-    // listen for the $ionicView.enter event:
-    //$scope.$on('$ionicView.enter', function(e) {
-    //});
+  function SidemenuController($scope, $ionicModal, $timeout) {
+    var vm = this;
 
     // Form data for the login modal
-    $scope.loginData = {};
+    vm.loginData = {};
 
     // Create the login modal that we will use later
     $ionicModal.fromTemplateUrl('modals/login.html', {
       scope: $scope
     }).then(function(modal) {
-      $scope.modal = modal;
+      vm.modal = modal;
+      $scope.vm = vm;
```

Comment:
I didn't liked this solution to pass ´´´$scope´´´ to the modal using the ´´´viewModel´´´ pattern, any good idea ? Should I just pass ´´´$scope´´´ in this case ?